### ISSUE TITLE
disk_user and disk_group are not applied correctly

### ISSUE THREAD

**Issue by laffinkippah** [AerisCloud/ansible-disk#12]
The task "Ensure the mount directory exists" should take disk_user and disk_group from item. Right now, it's always defaulting to root:

```
- name: "Ensure the mount directory exists"
  file:
    path: '{{ item.mount }}'
    owner: '{{ item.disk_user | default("root") }}'   #here
    group: '{{ item.disk_group | default("root") }}' #here
    state: directory
  with_items: '{{ disk_additional_disks }}'
  tags: ['disk']
```

**Comment by MiLk** [comment]
Would https://github.com/AerisCloud/ansible-disk/pull/13 solve your problem?

**Comment by laffinkippah** [comment]
It would. Thank you!

**Comment by MiLk** [comment]
Fixed by #13
